src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -123,7 +123,7 @@ private static FileInfo TryGetFileName(string basePath, string fileName)
             }
         }
 
-        [RavenAction("/studio/$", "GET", NoAuthorizationRequired = true)]
+        [RavenAction("/studio/$", "GET", RequiredAuthorization = AuthorizationStatus.UnauthenticatedClients)]
```

Comment:
We need to consider what kind of permissions is the minimum required to just show the auth error page.
Anything else should be `ValidUser`.